lib/html/pipeline/sanitization_filter.rb

```diff
@@ -25,27 +25,55 @@ class Pipeline
     #
     # This filter does not write additional information to the context.
     class SanitizationFilter < Filter
-      LISTS     = Set.new(%w(ul ol).freeze)
+      # These schemes are the only ones allowed in <a href> attributes by default.
+      ANCHOR_SCHEMES = ['http', 'https', 'mailto', :relative, 'github-windows', 'github-mac'].freeze
+
+      LISTS     = Set.new(%w(ol ul).freeze)
       LIST_ITEM = 'li'.freeze
 
+      DEF_LIST      = 'dl'.freeze
+      DEF_LIST_ITEMS = Set.new(%w(dt dd).freeze)
+
       # List of table child elements. These must be contained by a <table> element
       # or they are not allowed through. Otherwise they can be used to break out
       # of places we're using tables to contain formatted user content (like pull
       # request review comments).
-      TABLE_ITEMS = Set.new(%w(tr td th).freeze)
       TABLE = 'table'.freeze
-      TABLE_SECTIONS = Set.new(%w(thead tbody tfoot).freeze)
-
-      # These schemes are the only ones allowed in <a href> attributes by default.
-      ANCHOR_SCHEMES = ['http', 'https', 'mailto', :relative, 'github-windows', 'github-mac'].freeze
+      TABLE_SECTIONS = Set.new(%w(caption colgroup thead tbody tfoot).freeze)
+      TABLE_ITEMS = Set.new(%w(col tr td th).freeze)
 
       # The main sanitization whitelist. Only these elements and attributes are
       # allowed through by default.
       WHITELIST = {
         :elements => %w(
-          h1 h2 h3 h4 h5 h6 h7 h8 br b i strong em a pre code img tt
-          div ins del sup sub p ol ul table thead tbody tfoot blockquote
-          dl dt dd kbd q samp var hr ruby rt rp li tr td th s strike
+          # block elements (with mixed content)
```

Comment:
I see that when looking into my history.

But this is a very minor issue (and there's no need here to set a "non-interpolating" restriction on the syntaxic string that just specifies element names.
I did not see the "%w" restriction, it could just be changed to be interpolated (safer to maintain, and allows standard comments), or just by using the standard notation of arrays in [], whose members are explicitly strings.

My suggestion was to summarize all the missing HTML5 elements that are safe and summarize them in broad categories from the standard itself.